draft-fossati-cose-profiles.md

```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
+This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
 
 --- middle
 
 # Introduction
 
-This document lays a set of rules for how to define COSE {{-cose}} profiles.
+COSE {{-cose}} is not an end-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementation that want to interoperate need to agree on which subset of COSE features they will be using.
 
+This document provides a set rules for defining COSE profiles and registers a new COSE header parameter for in-band signalling of profile information.
```

Comment:
COSE {{-cose}} is not an end-to-end system with guaranteed interoperability.
 It is designed to serve a range of use cases and therefore it has a lot of options.
 In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.

This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.